### PR TITLE
cases: fix Versioning and split Data Registry

### LINKED ISSUE #3888: cases: split and complete Data Registry (combine with pipelines)
### Report

The current https://dvc.org/doc/use-cases/data-registry#building-registries is focused on `dvc add` / `.dvc` files workflow.

However, the same pattern can be applied using outputs of `dvc.yaml`, allowing users to introduce preprocessing steps in the Data Registry repo. 
This possibility is not obvious when following the current page.

UPDATE: Jumo to https://github.com/iterative/dvc.org/issues/3888#issuecomment-1236273489.

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Closes #3888

### CODE REVIEW

File: content/docs/sidebar.json [hunk 1]
```diff
@@ -76,9 +76,7 @@
     "source": "use-cases/index.md",
     "children": [
       {
-        "label": "Versioning Data and Models",
-        "slug": "versioning-data-and-model-files",
-        "source": "versioning-data-and-model-files/index.md",
+        "slug": "versioning-data-and-models",
         "children": [
```
Comment (jorgeorpinel) [inline content/docs/sidebar.json:80]: Fixed this (unrelated to #3888, sorry)

File: content/docs/use-cases/data-registry/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  Think "Git for cloud storage".
+- **Security**: DVC-controlled [remote storage] (e.g. Amazon S3) can be
+  configured to limit data access. For example, you can setup read-only
+  endpoints (e.g. an HTTP server) to prevent data deletions or alterations.
+
+[remote storage]: /doc/command-reference/remote
+
+👩‍💻 Intrigued? Try our [registry tutorial] to learn how DVC looks and feels
+firsthand.
+
```
Comment (jorgeorpinel) [inline content/docs/use-cases/data-registry/index.md:39]: Hmmm. Git thinks I created this new file but in fact it was just the first half of what I split. Nothing changed except this new link to its tutorial.

File: content/docs/use-cases/data-registry/tutorial.md [hunk 3]
```diff
@@ -1,44 +1,11 @@
...
-
-</admon>
+---
+title: 'Tutorial: Data Registry Basics'
+description:
+  'Get hands-on experience with building, using, and maintaining data registries
+  with DVC.'
+---
+
+# Tutorial: Data Registry Basics
```
Comment (jorgeorpinel) [inline content/docs/use-cases/data-registry/tutorial.md:8]: The 2nd half is in this new page now.

File: redirects-list.json [hunk 4]
```diff
@@ -47,10 +47,12 @@
...
   "^/doc/tutorial(/.*)?$                                                                  /doc/start",
   "^/doc/tutorials/get-started(/.*)?$                                                     /doc/start",
-  "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-model-files/tutorial",
+  "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-models/tutorial",
+  "^/doc/use-cases/versioning-data-and-model-files/tutorial$                              /doc/use-cases/versioning-data-and-models/tutorial",
   "^/doc/tutorials(/.*)?                                                                  /doc/start",
 
-  "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-model-files",
+  "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-models",
+  "^/doc/use-cases/versioning-data-and-model-files/?$                                     /doc/use-cases/versioning-data-and-models",
```
Comment (jorgeorpinel) [inline redirects-list.json:55]: This is part of the Versioning Data case URL fix. I'm updating existing redirects to final locations per https://github.com/iterative/dvc.org/pull/3772#discussion_r924066774.

**Review by jorgeorpinel — COMMENTED** [review]
Sorry, I thought fixing the existing Versioning Data and Models URL was a small change but due to link updates that takes most of the changes here. You can ignore most files (just link updates). Summary of relevant changes:

File: content/docs/use-cases/data-registry/tutorial.md [hunk 5]
```diff
@@ -166,7 +133,7 @@ scenarios such as a _model zoo_.
 
 See also the `dvc.api.read()` and `dvc.api.get_url()` functions.
 
-## Updating registries
+## Maintaining registries
```
Comment (shcheklein) [inline content/docs/use-cases/data-registry/tutorial.md:136]: Updating was better (and anyways, let's not change minot things please in an allready long PR)

**Review by shcheklein — APPROVED** [review]

File: content/docs/use-cases/data-registry/tutorial.md (hunk 5 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/data-registry/tutorial.md:136]: I forgot I changed that (to match the intro sentence).

```suggestion
## Updating registries
```

**Pull request merged**